src/main/java/reposense/model/RepoLocation.java

```diff
@@ -96,7 +63,11 @@ public boolean equals(Object other) {
         }
 
         RepoLocation otherLocation = (RepoLocation) other;
-        return this.location.equals(otherLocation.location);
+        return location.equals(otherLocation.location)
+                && repoName.equals(otherLocation.repoName)
+                && (organization == null ? otherLocation.organization == null
```

Comment:
perhaps can place the `?` at a new line to make it clearer?

Reply:
Thanks for the feedback. I will change this.